README.md

```diff
@@ -153,9 +155,16 @@ their Windows machine and use X11-forwarding to display the GUI.
     ```bash
     export DISPLAY=$(awk '/nameserver / {print $2; exit}' /etc/resolv.conf 2\>/dev/null):0
     export LIBGL_ALWAYS_INDIRECT=0
-    export MESA_GL_VERSION_OVERRIDE=3.3
     ```
 
+-----
+:warning: If you have this in your environment, you will need to remove it :warning:
```

Comment:
Reads a bit colloquially. Maybe: "In case you set `export MESA_GL_VERSION_OVERRIDE=3.3` in your environment, you will need to remove it."